alpha.py

```diff
@@ -54,14 +54,14 @@ def is_valid(self):
         TODO: implement value checks.
         """
         expected_types = {
-                self.author: str,
-                self.start_date: datetime.date,
-                self.alpha: BaseAlphaFormula,
-                self.target: tuple,
-                self.data_list: list
+            self.author: str,
+            self.start_date: datetime.date,
+            self.alpha: BaseAlphaFormula,
+            self.target: tuple,
+            self.data_list: list
         }
         return all(isinstance(variable, expected_type)
-                for variable, expected_type in expected_types.items())
+                   for variable, expected_type in expected_types.items())
```

Comment:
All these changes are not related to this PR. Can you remove these, and make them another PR, if you want?